biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/GenericGenbankHeaderFormat.java

```diff
@@ -38,6 +39,7 @@
 	private static final int HEADER_WIDTH = 12;
 	private static final String lineSep = "%n";
 	private String seqType = null;
+	private Boolean useOriginalHeader = false;
```

Comment:
Any specific reason not to use the primitive `boolean`?

Reply:
No, I have now fixed this